Index: src/assets/js/controllers/signUpController.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/assets/js/controllers/signUpController.js b/src/assets/js/controllers/signUpController.js
--- a/src/assets/js/controllers/signUpController.js	
+++ b/src/assets/js/controllers/signUpController.js	
@@ -21,9 +21,8 @@
     async #setupView() {
         this.#createSingInView = await super.loadHtmlIntoContent("html_views/signUp.html");
 
-
         this.#createSingInView.querySelector(".register-button").addEventListener("click",
-            (event) => this.#saveUser(event))
+            (event) => this.#saveUser(event));
 
     }
 
@@ -50,7 +49,7 @@
 
         } else {
             //user data opvragen database
-            this.#setSuccesfor(name);
+
             let data;
 
             try {
@@ -80,7 +79,7 @@
             this.#setErrorfor(email, "email moet een mail zijn!")
         } else {
             // database opvragen gegevens en checken of mail al bestaat
-            this.#setSuccesfor(email)
+            // this.#setSuccesfor(email)
 
             let data;
             try {
@@ -136,7 +135,12 @@
             alert("aanmelding is gelukt")
 
             // toevoegen aan database --------------------------
-            this.#signUpRepository.signUpUser(name.value, password.value, email.value)
+            const user = {
+                email: email,
+                username: username,
+                password: password
+            };
+            this.#signUpRepository.signUpUser(user)
 
         }
 
@@ -174,4 +178,5 @@
 
     }
 
-}
\ No newline at end of file
+}
+
Index: src/html_views/navbar.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/html_views/navbar.html b/src/html_views/navbar.html
--- a/src/html_views/navbar.html	
+++ b/src/html_views/navbar.html	
@@ -1,3 +1,6 @@
+<link rel="stylesheet" href="../assets/css/defaults.css">
+
+
 <nav class="navbar navbar-expand-lg navbar-dark static-top border-bottom">
     <div class="container">
         <i class="fa-regular fa-house-heart" src="../assets/img/logo.png"></i>
@@ -8,15 +11,33 @@
         </button>
         <div class="collapse navbar-collapse" id="navbarResponsive">
             <ul class="navbar-nav ml-auto">
+
+<!--                Home pagina -->
                 <li class="nav-item active">
                     <a class="nav-link" href="#" data-controller="welcome">Welcome</a>
                 </li>
-                <li class="loggedout">
+
+<!--             Verhalen pagina   -->
+                <li class="nav-item">
                     <a class="nav-link" data-controller="posts" href="#">Posts</a>
                 </li>
-                <li class="loggedout">
+
+<!--               Nav voor verhalen uploaden -->
+                <li class="nav-item">
                     <a class="nav-link" data-controller="upload" href="#">Upload</a>
                 </li>
+
+<!--                Instelling pagina -->
+                <li class="nav-item">
+                    <a class="nav-link" data-controller="settings" href="#">Settings</a>
+                </li>
+
+<!--              Support Pagina  -->
+                <li class="nav-item">
+                    <a class="nav-link" data-controller="support" href="#">Support</a>
+                </li>
+
+<!--                Account -->
                 <li class="loggedin">
                     <a class="nav-link" data-controller="login" href="#">Login</a>
                 </li>
@@ -24,7 +45,7 @@
                     <a class="nav-link" data-controller="logout" href="#">Logout</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" data-controller="signUp" href="#">signup</a>
+                    <a class="nav-link" data-controller="signIn" href="#">Sign in</a>
                 </li>
 
             </ul>
Index: src/assets/js/repositories/signUpRepository.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/assets/js/repositories/signUpRepository.js b/src/assets/js/repositories/signUpRepository.js
--- a/src/assets/js/repositories/signUpRepository.js	
+++ b/src/assets/js/repositories/signUpRepository.js	
@@ -20,6 +20,13 @@
         this.#networkManager.doRequest(this.#route,"POST", { username: username, password: password, email: email})
 
     }
-
+//     async signUpUser(username, email, password) {
+//         return await this.#networkManager.doRequest(${this.#route}/register, "POST", {
+//         "email": email,
+//             // "name": fullname,
+//             "username": username,
+//             "password": password
+//     });
+// // }
 
 }
\ No newline at end of file
Index: server/routes/signUpRoutes.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/routes/signUpRoutes.js b/server/routes/signUpRoutes.js
--- a/server/routes/signUpRoutes.js	
+++ b/server/routes/signUpRoutes.js	
@@ -18,8 +18,8 @@
             
             try{
                 const data = await this.#databaseHelper.handleQuery({
-                    query: "INSERT INTO users(username,password,email) values(?,?,?)",
-                    values: [req.body.username, req.body.password,req.body.email]
+                    query: "INSERT INTO users(username, password, email) values(?,?,?)",
+                    values: [req.body.username, req.body.password, req.body.email]
                 });
 
                 res.status(this.#errorCodes.HTTP_OK_CODE).json(data);
